lib/xcode.js

```diff
@@ -40,52 +40,6 @@ var cache,
  * what Watch Simulator. It's a mystery!
  */
 const simulatorDevicePairCompatibility = {
-	'>=6.2 <7.0': {             // Xcode 6.2, 6.3, 6.4
-		'>=8.2 <9.0': {         // iOS 8.2, 8.3, 8.4
-			'1.x': true         // watchOS 1.0
-		}
-	},
-	'7.x': {                    // Xcode 7.x
-		'>=8.2 <9.0': {         // iOS 8.2, 8.3, 8.4
-			'1.x': true         // watchOS 1.0
-		},
-		'>=9.0 <=9.2': {        // iOS 9.0, 9.1, 9.2
-			'>=2.0 <=2.1': true // watchOS 2.0, 2.1
-		},
-		'>=9.3 <10': {          // iOS 9.3
-			'2.2': true         // watchOS 2.2
-		}
-	},
-	'8.x': {                    // Xcode 8.x
-		'>=9.0 <=9.2': {        // iOS 9.0, 9.1, 9.2
-			'>=2.0 <=2.1': true // watchOS 2.0, 2.1
-		},
-		'>=9.3 <10': {          // iOS 9.x
-			'2.2': true,        // watchOS 2.2
-			'3.x': true         // watchOS 3.x
-		},
-		'10.x': {               // iOS 10.x
-			'2.2': true,        // watchOS 2.2
-			'3.x': true         // watchOS 3.x
-		}
-	},
-	'9.x': {                    // Xcode 9.x
```

Comment:
As of a week ago, we still support Xcode 9, so this would have to stay.

Secondly, I don't see why we need to remove the Xcode <9 lookup. It isn't hurting anything.

Reply:
It hurts me. And Xcode 9 is retained even for Xcode 12 support? Sorry but no one using Ti SDK 7 or higher uses Xcode 9 anymore.

Reply:
I hear ya, but technically removing support for anything, even deprecated old Xcodes, is a breaking change. Just leave the old versions alone.

Xcode 12 beta still extract...